ci/get_features.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+# Copyright 2020 JUUL Labs
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import configparser
+import argparse
+import os.path
+
+parser = argparse.ArgumentParser(description='Print features from a Cargo.toml.')
+parser.add_argument('infile', help='Input file to parse')
+
+args = parser.parse_args()
+if not os.path.isfile(args.infile):
+    print("File not found")
+    exit(1)
+
+config = configparser.ConfigParser()
```

Comment:
There is no requirement that Cargo.toml be a valid configparser file. I think this would be better done with a real toml parser, or we will possibly see strange errors if we use some of the flexibility of toml that isn't in configparser.

Reply:
Yeah, I did not want to add a new external dependency, but it's ok, I will update the code.